codebuild/samples/linux-smoke-tests.yml

```diff
@@ -15,6 +15,10 @@ phases:
       - $CODEBUILD_SRC_DIR/codebuild/samples/setup-linux.sh
       - $CODEBUILD_SRC_DIR/codebuild/samples/pubsub-linux.sh
       - $CODEBUILD_SRC_DIR/codebuild/samples/connect-linux.sh
+
+      # Currently dies in CI every time despite working perfectly fine locally - not sure why.
+      # TODO - fix/revisit this!
```

Comment:
Unless this is a pressing matter and someone is waiting for it, maybe we should wait on this merge to main until we get it working in CI. Otherwise this might become a task that just kind of falls behind. That's my opinion anyway, maybe worth discussing at standup.

Reply:
(posting here in case anyone outside if the team is following the PR) - Will continue to look into this and get it working with CI 👍